functions/__zgenom_init_cmd

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env zsh
+
+# 'initialize' a command by running 'exec' as well as saving it to ZGENOM_INIT_BUILDER
+# string that is used to create the init file
+function __zgenom_init_cmd() {
+  ZGENOM_INIT_BUILDER+="$*"$'\n'
+  eval "$*"
```

Comment:
In my testing everything seems to work.  I've done a visual inspection, checked that my modules are loading in the right order, I don't see any issues with commands, aliases, themes, etc.  

This should preserve the quoting behavior of the code without this change.  Please let me know if you see any issues

Reply:
using `eval` here preserves the functionality of users who always run `zgenom load ...` and never run `zgenom save`.